### PR TITLE
feat(console): implement get started page (client UI)

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

Implement "Get Started" page in Admin Console

image

image

## Linear Issue Reference

[LOG-2382](https://linear.app/silverhand/issue/LOG-2382/implement-get-started-page)

## Testing

- [x] Get Started page works

### CODE REVIEW

**Review by gao-sun — COMMENTED** [review]
i would like to hold until #777 is merged

File: packages/console/src/pages/GetStarted/index.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        <Button
+          className={styles.button}
+          type="outline"
+          size="large"
+          title="general.check_out"
+          onClick={() => {
+            console.log('lala');
+          }}
+        />
+      </Card>
```
Comment (gao-sun) [inline packages/console/src/pages/GetStarted/index.tsx:47]: nit: this can be a component as well

**Comment by demonzoo** [comment]
> overall looks good though.
> 
> nit: this title is kinda confusing. i thought it works from frontend to backend

Oops, my bad. This is client UI only, as including backend would greatly enlarge this PR. So I prefer to split them up

File: packages/console/src/pages/GetStarted/index.tsx (hunk 1 above)
Comment (demonzoo) [reply packages/console/src/pages/GetStarted/index.tsx:47]: Will refactor using data.map

**Comment by gao-sun** [comment]
would you like to rebase master?

**Comment by demonzoo** [comment]
> would you like to rebase master?

Sure, I have to. Doing this now

**Review by gao-sun — APPROVED** [review]

**Comment by demonzoo** [comment]
> #### Size Diff (Compare to `master`)
> **Total** ⚠️ 📈 +29.07 KB
> 
> Name	Diff
> packages/console/src/App.tsx	📈 +111 Bytes
> ,	packages/console/src/assets/images/cake.svg
> ,	packages/console/src/assets/images/crab.svg
> ,	packages/console/src/assets/images/drink.svg
> ,	packages/console/src/assets/images/frog.svg
> ,	packages/console/src/assets/images/grinning-face.svg
> ,	packages/console/src/assets/images/owl.svg
> ,	packages/console/src/pages/GetStarted/index.module.scss
> ,	packages/console/src/pages/GetStarted/index.tsx
> ,	packages/phrases/src/locales/en.ts
> ,	packages/phrases/src/locales/zh-cn.ts

@gao-sun Looks like we have to adjust the file size diff script

**Comment by gao-sun** [comment]
@demonzoo let's rebase

**Comment by demonzoo** [comment]
> @demonzoo let's rebase

Looks amazing!

**Pull request merged**